### PR TITLE
Setup project, install webpack, add linters.

### PR DESCRIPTION
**Pull request by koomekevo**
Add create-project, the first feature branch of the project that links to develop branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+- JavaScript
+- CSS
+- Webpack
+
+## Getting Started
+
+Clone the git repository using SSH or HTTP
+
+Next open terminal run `cd Leaderboard`
+
```
Comment (TedLivist) [inline README.md:17]: - [ ] A user cannot set up your project locally without running `npm install`. Kindly add it as a necessary command for setting up the project.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @koomekevo,

Good job so far!
✔️ Your app looks good
✔️ Neat code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (koomekevo) [reply README.md:17]: Thank you for your review. I've added `npm install` as a necessary command for setting up the project in the README.md

**Review by mikethreels — APPROVED** [review]
Hi Kevin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**